Dockerfile.assisted-installer

```diff
@@ -1,8 +1,16 @@
-FROM quay.io/centos/centos:centos8
-copy build/installer /
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 AS builder
+ENV GOFLAGS=-mod=mod
+WORKDIR /go/src/github.com/openshift/assisted-installer
+COPY . .
+RUN make installer
+
+FROM registry.svc.ci.openshift.org/ocp/4.7:base
```

Comment:
What's wrong with `quay.io/centos/centos:centos8` ? :)

Reply:
My reason for choosing this was to be more in-line with what we are doing downstream https://pkgs.devel.redhat.com/cgit/containers/assisted-installer/tree/Dockerfile?h=rhaos-4.6-rhel-8 
https://pkgs.devel.redhat.com/cgit/containers/assisted-installer-agent/tree/Dockerfile?h=rhaos-4.6-rhel-8

I know I have at least heard these debates rage on about which image to use. Any specific reason to avoid ocp 4.7 base? Should I pull in others to get feedback.